code.json

```diff
@@ -91,22 +91,22 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "8.1.0_RC1",
+    "version": "8.1.0_RC2",
```

Comment:
Can this be a separate code.json entry?

Reply:
Should we be appending for the code.json for every new release?

Reply:
Yes, it gets a little annoying but every release needs a code.json entry

Reply:
Np, I think I was just confused since when I've looked at previous release PRs, they would overwrite the existing code.json entry and also the current code.json entry only shows `8.0.0` entry but nothing prior.